### ISSUE TITLE
scons uninstall doesn't remove the Python package on Linux

### ISSUE THREAD

**Issue by bryanwweber** [Cantera/cantera#287]
After installing into the `--user` directory, `~/.local/`, running `scons uninstall` doesn't remove the `Cantera-2.2.0-py*.*.egg` directory from the `~/.local/lib/python*/site-packages` directory. I'm not sure if the uninstallation also doesn't remove this folder for an install into `/usr/local/lib/python*/{dist,site}-packages`.

**Comment by speth** [comment]
Strange. The behavior I get (which is also wrong) is that it leaves behind the compiled extension modules, e.g. `~/.local/lib/python2.7/site-packages/cantera/_cantera.so` and `~/.local/lib/python3.4/site-packages/cantera/_cantera.cpython-34m.so` but nothing else. I don't even have a `Cantera*.egg` directory after installing the module, just the `cantera` directory and the `Cantera*.egg-info` file under `site-packages`.

**Comment by bryanwweber** [comment]
I think which directory is present depends on whether we use `distutils` or `setuptools` to install. I originally noticed the `*.egg-info` file, but when I removed it manually and checked again after the next install, I didn't see it. But, I noticed it this morning when I was working on my `setuptools` patches, so I suspect the files that you see are more likely to be correct based on the current HEAD.
